iroha/src/queue.rs

```diff
@@ -4,11 +4,28 @@ use std::time::Duration;
 
 use crossbeam_queue::ArrayQueue;
 use dashmap::{mapref::entry::Entry, DashMap};
-use eyre::Result;
+use eyre::{Report, Result};
 
 use self::config::QueueConfiguration;
 use crate::{prelude::*, wsv::WorldTrait};
 
+/// Queue error.
+#[derive(thiserror::Error, Debug)]
+pub enum Error {
+    /// Transaction expired.
+    #[error("Transaction expired.")]
+    TransactionExpired,
+    /// Transaction is already in blockchain.
+    #[error("Transaction is already in blockchain.")]
+    AlreadyInBlockchain,
+    /// Failure during signature condition execution.
+    #[error("Failure during signature condition execution: `{0}`")]
```

Comment:
Should we print internal error here?

Reply:
To my mind yes, it will tell what is wrong with accounts signature condition configuration.

Reply:
It will help our client developers and other developers building on Iroha a lot

Reply:
We discussed in telegram: it is unnecessary as it will print it due to having this error as source.

Reply:
After some more research we found a problem with this for our simple torii into response http converter for errors, as it doesn't follow the source. Planning to post an issue for this.